packages/flutter/test/foundation/annotations_test.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  test('test DocumentationIcon object', () {
```

Comment:
It looks like this test is testing the `url` property of the annotation and/or the constructor. Can we update the test name to reflect that?